artemis-docker/Dockerfile-ubuntu-11-jre

```diff
@@ -17,7 +17,30 @@
 
 # ActiveMQ Artemis
 
+ARG CURRENT_VERSION=2.17.0
+
+FROM eclipse-temurin:11-jdk as builder
+ARG CURRENT_VERSION
+
+ENV VERSION=$CURRENT_VERSION
+
+RUN apt update -y && apt upgrade -y && apt install tree curl -y
+
+ADD ./prepare-docker.sh /bin/prepareDocker
+WORKDIR /root/artemis-build
+COPY ./Dockerfile-alpine .
```

Comment:
Shouldn't this be `COPY ./Dockerfile-ubuntu-11-jre .`?

Reply:
It doesn't matter as I don't need the Dockerfile in my multistage build. So I removed the line in the prepareDocker script that checks if a Dockerfile* is available. It still works as expected.